service-commands/scripts/userClockValues.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Find the creator nodes associated with a user and their respective clock values.
+ *
+ * Script usage: node userClockValues.js <size> <outFile>
+ */
+const axios = require('axios')
+
+const discoveryProvider = 'https://discoveryprovider.audius.co'
+
+async function run() {
+  try {
+    const { handle } = parseArgs()
+
+    const { wallet, creator_node_endpoint: creatorNodeEndpoint } = (
+      await axios({
+        url: `/v1/full/users/handle/${handle}`,
+        method: 'get',
+        baseURL: discoveryProvider
+      })
+    ).data.data[0]
+
+    const creatorNodes = creatorNodeEndpoint
+      ? creatorNodeEndpoint.split(',')
+      : []
+
+    creatorNodes.forEach(async creatorNode => {
+      const { clockValue } = (
+        await axios({
+          url: `/users/clock_status/${wallet}`,
+          method: 'get',
+          baseURL: creatorNode
+        })
+      ).data
+
+      console.log(creatorNode, clockValue)
+    })
+  } catch (err) {
+    console.error(err.message)
```

Comment:
could we update this error message to print out `err` instead of `err.message`? there might be a scenario where `err.message` doesn't exist. also, could we make these error messages more descriptive and maybe wrap the try/catches around the axios requests?
e.g. 
- "Could not get wallet and endpoint from discovery node ${discoveryProvider}:  ${e}"
- "Could not fetch clock values for handle ${handle} at endpoint ${creatorNode}: ${e}"

Reply:
Thanks, I made the error handling look much nicer now. 😁 i